src/components/react-admin/CustomToolbar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { withRouter, Link } from 'react-router-dom';
+import { SaveButton, Toolbar, DeleteWithConfirmButton, Button } from 'react-admin';
+
+import { withStyles } from '@material-ui/core/styles'; // eslint-disable-line import/no-extraneous-dependencies
+
+
+import IconClose from '@material-ui/icons/Close'; // eslint-disable-line import/no-extraneous-dependencies
+
+import compose from '../../utils/compose';
+
+const styles = {
+  toolbar: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+  },
+  filler: {
+    flex: 1,
+  },
+};
+
+
+const PostCreateToolbar = ({
```

Comment:
Let's add some documentation

Reply:
You're so damn right ! Done. (Also the naming was copypasted).